kolibri/core/assets/src/core-app/mediator.js

```diff
@@ -82,13 +93,16 @@ module.exports = class Mediator {
     this._executeCallbackBuffer(kolibriModule);
     logging.info(`KolibriModule: ${kolibriModule.name} registered`);
     this.emit('kolibri_register', kolibriModule);
-    if (this._ready) {
-      kolibriModule.ready();
-    } else {
-      this._eventDispatcher.$once('ready', () => {
+    const ready = () => {
```

Comment:
Is this rewriting this function into ES6, plus adding `kolibriModule.ready()`? Or is there a deeper algorithm change here?